### PR TITLE
Disable warnings and update the SDK version

### PR DESCRIPTION
**Pull request by nirinchev**
Setting some groundwork for a beta release later this week

### CODE REVIEW

File: Realm/Realm.Fody/Common/Analytics.cs [hunk 1]
```diff
@@ -118,7 +116,8 @@ private string JsonPayload
                     .Replace("%SYNC_ENABLED%", _isSyncEnabled.ToString())
 
                     // Version of weaver is expected to match that of the library.
-                    .Replace("%REALM_VERSION%", Assembly.GetExecutingAssembly().GetName().Version.ToString())
+                    // TODO: temp - remove that when we're out of beta
+                    .Replace("%REALM_VERSION%", "10.0.0-beta.1") // Assembly.GetExecutingAssembly().GetName().Version.ToString())
```
Comment (papafe) [inline Realm/Realm.Fody/Common/Analytics.cs:120]: Shouldn't this be 10.2.0-beta.1?

File: Realm/Realm/Handles/AppHandle.cs [hunk 2]
```diff
@@ -175,7 +175,9 @@ static unsafe AppHandle()
 
             var platform = InteropConfig.Platform;
             var platformVersion = RuntimeInformation.OSDescription;
-            var sdkVersion = InteropConfig.SDKVersion.ToString(3);
+
+            // TODO: temp - remove that once we're out of beta
+            var sdkVersion = "10.0.0-beta.1"; // InteropConfig.SDKVersion.ToString(3);
```
Comment (papafe) [inline Realm/Realm/Handles/AppHandle.cs:180]: Same question as before

File: Realm/Realm.Fody/Common/Analytics.cs (hunk 1 above)
Comment (nirinchev) [reply Realm/Realm.Fody/Common/Analytics.cs:120]: Doh 🤦‍♂️ apparently the sun was too much for me yesterday 😅

**Review by LaPeste — APPROVED** [review]
Look forward to the release :)

File: Realm/AssemblyInfo.props [hunk 3]
```diff
@@ -1,7 +1,7 @@
 <Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <Product>Realm .NET</Product>
-    <VersionPrefix>10.5.0</VersionPrefix>
```
[the remark this answers is not in the document]
Comment (nirinchev) [reply Realm/AssemblyInfo.props:4]: Nope, we never released anything with 10.5.

**Pull request merged**